server/server.js

```diff
@@ -10,7 +10,7 @@ const envConfig = dotenv.config();
 const dotenvExpand = require("dotenv-expand");
 dotenvExpand.expand(envConfig);
 
-const PORT = process.env.PORT || 5000; // Define the port number, use environment variable if available
```

Comment:
I suggest to keep fallback as 5000 as you can always change it on your local .env

Reply:
Still outstanding

Reply:
The port has been changed back to 5000.